runelite-client/src/main/java/net/runelite/client/plugins/chatfilter/ChatFilterPlugin.java

```diff
@@ -321,7 +321,8 @@ String censorMessage(final String username, final String message)
 	{
 		String strippedMessage = jagexPrintableCharMatcher.retainFrom(message)
 			.replace('\u00A0', ' ');
-		String strippedAccents = StringUtils.stripAccents(strippedMessage);
+		boolean shouldStripAccents = !strippedMessage.contains("(?U)");
```

Comment:
`strippedMessage` is the message from the player and not the configured pattern, so this doesn't work